### PR TITLE
RateLimiterDynamo: Add key to skip querying ttl

### PR DESCRIPTION
**Pull request by ollyfg**
TLDR: Adds an option allowing users to skip the check for TTL settings when using the `RateLimiterDynamo`.

When assessing this library for use in a serverless environment (in this case AWS Lambda), we noticed that there was a significant overhead every time we instantiated a new rate limiter (so every time a new lambda started). This was due to `RateLimiterDynamo`'s behaviour which is to query the table to make sure that the TTL has been set up correctly.

While this is useful behavior on a long-running server, it doesn't make much sense in a serverless environment - once we've validated that the TTL is indeed set up correctly it doesn't make sense to keep checking it again and again and again.

I propose a new option which disables this check, in much the same way that you can disable `RateLimiterDynamo`'s built-in table-creation logic.

### CODE REVIEW

**Comment by animir** [comment]
@ollyfg Hello, thank you! This looks great.
FYI, I'll add a new type for `ttlManuallySet` attribute to `index.d.ts`.

**Comment by animir** [comment]
Released in v `5.0.4`.
